pytri/__init__.py

```diff
@@ -248,6 +248,7 @@ def imshow(
             self,
             data,
             position={"x": 0, "y": 0, "z": 0},
```

Comment:
[The test failure](https://circleci.com/gh/iscoe/pytri/158) makes a good point — dict arguments are a bad idea. (Explanation [here](https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python)). Not your fault, since this existed in the code already, but would you mind making the fixing change in your PR?

* Change default arg to `None` for both pos and rot
* Add a `if position is None: position = { ... }` (and same for rotation) to the fn body

This seems like as good a place as any to make our code less buggy! :)

Reply:
Happy to make the change! Thanks for the pointers!

You confirmed for sure that with the `rotation.set(...)` added in, you still get images rendering? 99% certain it's my environment, but don't want to break things.